src/ui/pages/usersList/UsersList.tsx

```diff
@@ -159,7 +159,9 @@ export const UsersList: React.FC<UserListProps> = ({
 				src={LOGO_ICON_LIGHT}
```

Comment:
Unrelated to this file but we need to update the version of the package and changelog